www/css/themes/night.css

```diff
@@ -0,0 +1,5 @@
+body {
```

Comment:
I deleted light theme since it was 99% like the default one.

In my opinion, the gray theme doesn't look too good, so I would personally remove it.

The night and the blackout theme are very similar, so my opinion would be to keep only one of them.

Reply:
I didn't really like the gray either but it was an option. If i had to choose one i would do the night theme as i like it better than the blackout